packages/maestro/src/plugin/twig.js

```diff
@@ -151,6 +175,15 @@ const plugin = (options = {}) => {
           frameworkInclude = `import React from 'react'`;
           frameworkTransform = `const frameworkTransform = (html) => React.createElement('div', {dangerouslySetInnerHTML: {'__html': html}});;`;
         }
+
+        let isDynamic = false;
+
+        if (options.dynamics) {
```

Comment:
Would this work if a component that is specified in dynamics is used standalone?